twitch-chat-reader/src/listener.py

```diff
@@ -78,7 +78,7 @@ async def connect(self) -> None:
         await self.send(f"NICK {self.bot_name}")
         await self.send(f"JOIN #{self.channel}")        
         await self.send(f"CAP REQ :twitch.tv/tags")
-        await self.send(f"PRIVMSG #{self.channel} :I'm listening!")
+        # await self.send(f"PRIVMSG #{self.channel} :I'm listening!")
```

Comment:
please uncomment this because I didn't want a bot talking in someone else's room